pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-content.adoc

```diff
@@ -0,0 +1,22 @@
+[id="importing-(ContentTerm}_{context}"]
+
+= Importing {ContentTerm}
+
+[role="_abstract"]
+You can import modules and assemblies into {ProductShortName} from either GitHub or GitLab.
+
+.Prerequisites
+
+//* You are assigned a publisher or author role in {ProductShortName}.
+* You are logged in.
+* You obtained the Git repository URL for the {ContentTerm} you want to import (For example, https://github.com/openshift/openshift-docs).
+
+.Procedure
+
+. Navigate to the link:{LinkToGitImportPage}[Git Import page].
+. Enter the Git repository URL for the {ContentTerm} you want to import.
+. By default, this procedure imports the "master" branch of the {ContentTerm} repository, but you can enter a specific branch to import instead if you prefer.
```

Comment:
I don't understand this line. How do you import master? How do you import a branch?